kolibri2zim/scraper.py

```diff
@@ -256,12 +282,10 @@ def funnel_from_s3(self, file_id, path, checksum, preset):
 
         # add to zim
         with self.creator_lock:
-            self.creator.add_item(
-                StaticItem(
-                    path=path,
-                    fileobj=fileobj,
-                    mimetype=preset.mimetype,
-                )
+            self.creator.add_item_for(
+                path=path,
+                content=fileobj.read(),
```

Comment:
Really good to know.
I rewrote it so that the type checker does not complains, but there are typing issues in zimscraperlib are this, I opened an issue: https://github.com/openzim/python-scraperlib/issues/109